src/test/java/org/folio/test/support/ActionsAPITests.java

```diff
@@ -23,7 +24,8 @@ public static Account verifyAccountAndGet(ResourceClient accountsClient, String
       .body("remaining", is((float) amount.toDouble()))
       .body("status.name", is(statusName))
       .body("paymentStatus.name", is(expectedPaymentStatus))
-      .body("metadata.updatedDate", notNullValue());
+      .body("metadata.updatedDate", notNullValue())
+      .body("dateClosed", (statusName== "Open" ? nullValue() : notNullValue()));
```

Comment:
Please check that dateClosed is correct

Reply:
At this point I can't check the exact date, as the date is updated and I lose the exact second. However, I checked the log and it works fine, that's why I only set it to whether it is null or not